LoopKitUI/HorizontalSizeClassOverride.swift

```diff
@@ -20,4 +20,12 @@ public extension HorizontalSizeClassOverride {
             return .regular
         }
     }
+        
+    var isDisplayNarrow: Bool {
```

Comment:
I don't see this being used anywhere? I would like to see where this is used.

Hardcoded screen boundaries to be viewed with suspicion, and only used as a last resort. The existence of the function above this one is something I tried many other workarounds before resorting to.

Also, somewhat unrelated, but it looks like when this file was brought in from DashKit, it was turned into a public extension?  We should not be exporting extensions to platform types from LoopKit.

Reply:
This is used in the `StatusTableViewController` and `StatusViewController` (See here: https://github.com/tidepool-org/Loop/pull/249). 

I'm not aware of an other way to determine if the device display is considered narrow. Please let me know if I overlooked something.

Reply:
Creating a generic and app-wide "display is narrow" concept based on display width is problematic because different views will have different requirements as to when they need to adjust layout.  What is narrow for one view might be fine for another view.

It would be better if the view reacts to the space it is given. You can use GeometryReader for this in SwiftUI views, or other techniques if UIKit.  And the cutoff constraints for this decision should be local to the view.

Reply:
OK. I'll make it local to the view.